### PR TITLE
feat: added new branch, layout with header, footer and mobi navbar

### PR DESCRIPTION
**Pull request by Kamkoz93**

### CODE REVIEW

File: src/app/components/footer/footer.component.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  encapsulation: ViewEncapsulation.Emulated,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FooterComponent {
+  readonly categories$: Observable<ProductCategoryModel[]> =
+    this._productCategoriesService.getAllCategories().pipe(shareReplay(1));
+
+  readonly stores$: Observable<StoreModel[]> = this._storesService
+    .getAllStores()
+    .pipe(shareReplay(1));
```
Comment (lwalik) [inline src/app/components/footer/footer.component.ts]: Do you need this? If you use it only once then it is unnecessary

File: src/app/components/header/header.component.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+                                aria-expanded="false" aria-label="Toggle navigation"> <span
+                                    class="icon-bar top-bar mt-0"></span> <span class="icon-bar middle-bar"></span>
+                                <span class="icon-bar bottom-bar"></span>
+                            </button>
+                        </div>
+                    </div>
+                    <div class="navbar-collapse collapse" id="navbarSupportedContent">
+                        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                            <li *ngFor="let category of categories$ | async as categoryList" class="nav-item">
```
Comment (lwalik) [inline src/app/components/header/header.component.html]: Why you added "as categoryList" ?

File: src/app/components/header/header.component.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            <div class="container px-0 px-md-3">
+                <div class="offcanvas offcanvas-start p-4 p-lg-0">
+                    <div class="d-flex justify-content-between align-items-center mb-2 d-block d-lg-none"> <a
+                            routerLink="/"><img src="/assets/images/logo/freshcart-logo.svg"
+                                alt="eCommerce HTML Template"></a> <button type="button" class="btn-close"
+                            aria-label="Close"></button>
+                    </div>
+                    <div class="d-block">
+                        <ul class="nav col-12 col-lg-auto me-lg-auto mb-2 mb-md-0">
+                            <li *ngFor="let category of categories$ | async as categoryList"><a
```
Comment (lwalik) [inline src/app/components/header/header.component.html]: With "as categoryList" as above.

You subscribe to categories$ twice. You should do it only once

File: src/app/components/footer/footer.component.ts (hunk 1 above)
Comment (Kamkoz93) [reply src/app/components/footer/footer.component.ts]: right on, fixed it.

File: src/app/components/header/header.component.html (hunk 2 above)
Comment (Kamkoz93) [reply src/app/components/header/header.component.html]: My mistake, I have refactored this code from different code block and I havent brushed it entirely.  Fixed it.

File: src/app/components/header/header.component.html (hunk 3 above)
Comment (Kamkoz93) [reply src/app/components/header/header.component.html]: Fixed it by subscribing to the categories in heder tag,

**Review by lwalik — APPROVED** [review]

**Pull request merged**